src/screens/Home/components/SignUp/index.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Don't disable a rule for the entire file if you can avoid it. Do it in a specific block of code:

```
/* eslint-disable @typescript-eslint/naming-convention */
type InputsValues = {
  ...
}
/* eslint-enable @typescript-eslint/naming-convention */
```